src/qt/editor/inviwomainwindow.cpp

```diff
@@ -184,6 +186,28 @@ class MenuKeyboardEventFilter : public QObject {
     std::vector<Menu> menus_;
 };
 
+namespace detail {
+
+auto getWidgetProcessors(InviwoApplication* app) {
+    auto widgetProcessors = util::copy_if(app->getProcessorNetwork()->getProcessors(),
+                                          [](const auto p) { return p->hasProcessorWidget(); });
+    std::sort(widgetProcessors.begin(), widgetProcessors.end(),
+              [](auto a, auto b) { return iCaseLess(a->getDisplayName(), b->getDisplayName()); });
+    return widgetProcessors;
+}
+
+auto getFloatingDockWidgets(InviwoMainWindow* win) {
+    auto dockWidgets = util::copy_if(win->findChildren<InviwoDockWidget*>(),
+                                     [](const auto p) { return p->isFloating(); });
+    std::sort(dockWidgets.begin(), dockWidgets.end(), [](auto a, auto b) {
```

Comment:
Why are these sorted?

Reply:
They were initially only used for listing all the windows in the window menu